indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -162,8 +220,22 @@ sealed class Asn1Element(
                 return derEncoded
             }
 
+            /**
+             * Convenience helper to easily construct implicitly tagged elements.
+             * Shorthand for `Tag(tagValue, constructed=false, tagClass=TagClass.CONTEXT_SPECIFIC)
+             */
+            fun ULong.toImplicitTag() =
```

Comment:
Is this not duplicating the (more explicit, pun not intended) `long withClass CONTEXT_SPECIFIC`?

Reply:
it is, but you will need this all over the place when dealing with X.509

Reply:
then maybe a `Asn1.Tag(ULong)` constructor?

Reply:
that would cause more confusion: is it implicit? should I use that when invoking Asn1Tagged? etc…

I do get that both structures and primitives may be tagged implicitly.
Still, this will be used and the function name combined with the API docs leaves little to be desired.
Build some X.509 structures. implicitly tagged primitives are just so common, that this shorthand is warranted.

Reply:
`Asn1.ImplicitTag(ULong, tagClass = TagClass.CONTEXT_SPECIFIC)`?

Reply:
(i'm especially concerned about polluting the IDE auto-complete with "toImplicitTag", where it is not immediately clear what context "implicit tag" is even referring to... and if you call it "toImplicitAsn1Tag" the constructor is the same length)

Reply:
> `Asn1.ImplicitTag(ULong, tagClass = TagClass.CONTEXT_SPECIFIC)`?
> 
> `42u.toImplicitTag()` is hardly shorter than `Asn1.ImplicitTag(42u)`

I like that! done (also for explicit tag)